apis/core/config/crd/bases/core.tanzu.vmware.com_readinessproviders.yaml

```diff
@@ -92,6 +92,24 @@ spec:
                   - name
                   type: object
                 type: array
+              serviceAccount:
```

Comment:
can you name this field `serviceAccountRef` as suggested in Kubernetes API conventions doc https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#object-references-examples

Reply:
Done